A36-xds-for-servers.md

```diff
@@ -68,24 +68,25 @@ XdsServerCredentials.
 
 To serve RPCs, the XdsServer must have its xDS configuration, provided via a
 Listener resource and potentialy other related resources. When its xDS
-configuration is unavailable the server must be in a "not serving" mode. This is
-ideally implemented by not `listen()`ing on the port. If that is impractical an
-implementation may be `listen()`ing on the port, but it must also `accept()` and
-immediately `close()` connections, making sure to not send any data to the
-client (e.g., no TLS ServerHello nor HTTP/2 SETTINGS). With either behavior,
-client connection attempts will quickly fail and clients would not perform
-further attempts without a backoff. Load balancing policies like `pick_first`
-would naturally attempt connections to any remaining addresses to quickly find
-an operational backend. However, the `accept()`+`listen()` approach will be
-improperly detected as server liveness for TCP heath checking.
+configuration is missing or unknown the server must be in a "not serving" mode.
```

Comment:
I think this still isn't very clear to readers who don't already know what we mean.  How about saying "does not exist or has not yet been received"?